### PR TITLE
english quick start update for nette 2.3 

### PR DESCRIPTION
**Pull request by Cactucs**
Translation of pull #251 (based on issue #247).

### CODE REVIEW

File: en/quickstart/home-page.texy [hunk 1]
```diff
@@ -66,6 +80,9 @@ First, we need to tell our application which database to use. All the configurat
 .[note]
 Be aware of indenting while editing this file. [NEON |http://ne-on.org] format accepts both spaces and tabs but not both together! Configuration file in the Sandbox uses tabs as default.
 
+.[tip]
+Whole configuration including is stored in `app/confing/` in files `confing.eon` and `config.local.neon`.
```
Comment (Majkl578) [inline en/quickstart/home-page.texy]: typo `.eon`

File: en/quickstart/home-page.texy [hunk 2]
```diff
@@ -116,47 +133,41 @@ There is a generic template for the whole page (called *layout*, with header, st
...
-This line includes a named block called `content`, which is defined in the specific template.
-
-You can try to refresh your browser now, even though you haven’t created the template yet. Nette will display an error message:
-
-/--
-Page not found. Missing template '…/app/presenters/templates/Homepage/default.latte'.
-\--
-
-Now, create the missing file and insert the minimum required code:
+Code `{include content}` inserts to main template a block named `content`. You can define it in templates of each view. In this case we will edit file `app/presenters/templates/Hompeage/default.latte` like this:
```
Comment (Majkl578) [inline en/quickstart/home-page.texy]: > `{include content}` inserts a block named `content` into the main template.

File: en/quickstart/home-page.texy [hunk 3]
```diff
@@ -116,47 +133,41 @@ There is a generic template for the whole page (called *layout*, with header, st
...
 
 /--html
 {block content}
 \--
 
-It defines the *content* block, which will be inserted into the layout. If you refresh the browser, you’ll see an empty page (with only HTML header and footer).
+It defines the *content* [ block | default-macros#toc-bloky ], which will be inserted into the layout. If you refresh the browser, you’ll see a page with text "Hello word" (in source code also with HTML header and footer defined in `@layout.latte`).
 
-Let’s display the blog posts - add the following code to the template, after the block definition:
+Let’s display the blog posts - we will edit template like this:
```
Comment (Majkl578) [inline en/quickstart/home-page.texy]: the template?

File: en/quickstart/single-post.texy [hunk 4]
```diff
@@ -35,7 +36,33 @@ class PostPresenter extends BasePresenter
 
 The `renderShow` method requires one argument - the ID of the post to be displayed. Then, it loads the post from the database and passes the result to the template.
 
-In the `app/presenters/templates/Homepage/default.latte` template, we’ve already created a link to the `Post:show` action, for which a template doesn’t exist yet. You can try to open a post in your browser using the link in the list on the home page. [Tracy | /debugging] will show an error, because the `app/presenters/templates/Post/show.latte` template has not yet been created (if you see 404 error, you probably need to enable mod_rewrite at your webserver). Let’s add that one now:
+To the `app/presenters/templates/Homepage/default.latte` template we`ll add a link to action `Post:show`.
```
Comment (Majkl578) [inline en/quickstart/single-post.texy]: the action?

File: en/quickstart/single-post.texy [hunk 5]
```diff
@@ -35,7 +36,33 @@ class PostPresenter extends BasePresenter
...
+
+/--html
+...
+<h2><a href="{link Post:show $post->id}">{$post->title}</a></h2>
+...
+\--
+
+Macro `{link}` generates URL address which points to action `Post:show`. This macro also forwards ID of the post as argument.
```
Comment (Majkl578) [inline en/quickstart/single-post.texy]: the action?

File: en/quickstart/single-post.texy [hunk 6]
```diff
@@ -35,7 +36,33 @@ class PostPresenter extends BasePresenter
...
+...
+<h2><a n:href="Post:show $post->id">{$post->title}</a></h2>
+...
+\--
+
+Atribute `n:href` is alias for `{link}` macro.
+
+
+
+For Post:show action does not exists template. We can open link to this post. [Tracy | /debugging] will show error, why `app/templates/Post/show.latte` doesn't exists. If you see any other error reportm you probably have to turn on mod_rewrite at your webserver.
```
Comment (Majkl578) [inline en/quickstart/single-post.texy]: >  The template for `Post:show` action does not yet exist.

> an error

typo in `reportm`

> doesn't exist

> in your webserver

**Comment by Cactucs** [comment]
Fixed!

**Pull request closed without merging**